jaxrs-clients/src/test/java/com/palantir/remoting/jaxrs/FailoverFeignTargetTest.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.remoting.jaxrs;
+
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.startsWith;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.fail;
+import static org.mockito.Matchers.anyInt;
+import static org.mockito.Mockito.inOrder;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.google.common.collect.ImmutableList;
+import com.palantir.remoting.http.BackoffStrategy;
+import feign.RetryableException;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import okhttp3.mockwebserver.MockResponse;
+import okhttp3.mockwebserver.MockWebServer;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.InOrder;
+
+
+public final class FailoverFeignTargetTest {
```

Comment:
so this would be a pretty big refactor of the tests if we want to maintain similar coverage (and under the assumption that we'll soon delete http-clients). I'd prefer to settle on the API, get a version out, then refactor tests so they make more sense.